lib/App.js

```diff
@@ -2263,7 +2273,20 @@ class App {
   }
 
   static hasHomeyCompose({ appPath }) {
-    return fs.existsSync(path.join(appPath, '.homeycompose'));
+    const hasComposeFolder = fs.existsSync(path.join(appPath, '.homeycompose'));
+
+    if (
+      hasComposeFolder &&
+      !this.__warnedAboutMissingHomeyComposeManifest &&
+      !fs.existsSync(path.join(appPath, '.homeycompose/app.json'))
+    ) {
+      Log(colors.yellow('⚠ Warning: Could not find a Homey Compose app.json manifest!'));
+      Log(colors.yellow('Using the generated app.json in the root of your app is supported for now,'));
+      Log(colors.yellow('but it is recommended to move your manifest to the .homeycompose/ folder.'));
```

Comment:
```suggestion
      Log(colors.yellow('⚠ Warning: Could not find a Homey Compose app.json manifest!'));
      Log(colors.yellow('Using the generated app.json in the root of your app is supported for now,'));
      Log(colors.yellow('but it is recommended to move your manifest to .homeycompose/app.json.'));
```

Reply:
can't update in master so https://github.com/athombv/node-homey/pull/167